src/runtime/mini_ion.h

```diff
@@ -0,0 +1,18 @@
+#ifndef HALIDE_RUNTIME_ION_H
+#define HALIDE_RUNTIME_ION_H
+
+#include "HalideRuntime.h"
+
+namespace Halide { namespace Runtime { namespace Internal { namespace Ion {
+
+// Allocate an ION buffer, and map it, returning the mapped
```

Comment:
Please define what ION/Ion/ion actually means somewhere. Is it an Android thing? An HVX thing? Something else? I have no idea. Also, please use a consistent capitalization style for it everywhere.